server/controllers/user/getAdminData.js

```diff
@@ -0,0 +1,13 @@
+const { getAdminBudget } = require('../../database/queries/index');
+
+const adminData = async (req, res, next) => {
+  try {
+    const { rows: data } = await getAdminBudget();
+    res.status(200).json({ statusCode: 200, data });
+  } catch (err) {
+    next(err);
+  }
+  res.json({ msg: 'admin data' });
```

Comment:
For what this response is sent ?

Reply:
for test controller and forget to delete :)